src/do-you-want-to-stop.py

```diff
@@ -1,3 +1,10 @@
 
 # You should write a loop around this question, and keep asking until
 # the answer is "yes" (lower case).
+from tkinter import YES
+
+
+a = YES
+while a = a:
```

Comment:
This is not going to work. First off, `a = a` is an assignment and not an expression (for an expression, you need `a := a` but you shouldn't use that here), so syntactically there is a problem.

Second, whatever `a` is, so is `a`, so is `a`, so with `a == a` you would have an expression that is always `True`.